packages/destination-actions/src/destinations/fullstory/__tests__/fullstory.test.ts

```diff
@@ -126,10 +131,14 @@ describe('FullStory', () => {
       expect(JSON.parse(response.options.body as string)).toEqual({
         segmentAnonymousId_str: anonymousId,
         email,
+        // TODO(nate): See if we can eliminate duplicate email_str and name_str data based on mapping config.
```

Comment:
I briefly investigated and didn't see that we have access to the mapping configuration in the `perform` action which would seemingly limit our ability to verify that e.g. `name` is mapped to `displayName` such that we could avoid including `name_str` in the custom vars.